### PR TITLE
updated bacalahau server side to work with single IPFS CID

### PR DESCRIPTION
**Pull request by stan36**

### CODE REVIEW

**Review by ArnobChowdhury — CHANGES_REQUESTED** [review]
Just one comment on using the `setTimeout`. Please consider using [execSync](https://nodejs.org/api/child_process.html#child_processexecsynccommand-options).

File: server/routes/bacalhau.js [hunk 1]
```diff
@@ -1,38 +1,23 @@
...
-  const rawArray = body.completedJobsArray;
-  let newCommand = "";
-
-  for (var i = 0; i < rawArray.length; i++) {
-    console.log(rawArray[i]);
-    newCommand = newCommand + " " + rawArray[i] + " ";
-    console.log(newCommand);
-  }
+  console.log(body.completedJobsArray);
+  const cid = body.completedJobsArray;
```
Comment (ArnobChowdhury) [inline server/routes/bacalhau.js:8]: Can this be `body.cid` instead of `body.completedJobsArray`?

Comment (stan36) [reply server/routes/bacalhau.js:8]: yea

File: server/routes/bacalhau.js [hunk 2]
```diff
@@ -1,38 +1,23 @@
-var express = require('express');
+var express = require("express");
 var router = express.Router();
-const { exec, spawn } = require('node:child_process');
+const { exec, spawn } = require("node:child_process");
```
[the remark this answers is not in the document]
Comment (stan36) [reply server/routes/bacalhau.js]: I went ahead and added execsync. After looking at the docs I reralized that was not the one I tried to use that caused me issues. I didnot remove the timeout yet as I will need to set things up on my end to test it all out.

**Review by ArnobChowdhury — APPROVED** [review]
Let's get it merged.

**Pull request merged**